README.md

```diff
@@ -21,12 +23,28 @@ buildscript {
 apply plugin: 'com.android.application'
 apply plugin: 'redex'
 
-// Optional: set which passes are run
-redex.passes = ["ReBindRefsPass", ..., "ShortenSrcStringsPass"]
+// Optional: configure redex command line arguments
+redex {
+    // These are example arguments, fill them with your specific arguments
+    configFile = 'redex.config'
+    // `passes` is shorthand for a pass list instead of a config file
+    // passes = ['ReBindRefsPass', ..., 'ShortenSrcStringsPass']
+    proguardConfigFiles = ['common_proguard.pro', 'my_app_proguard.pro']
+    proguardMapFile = 'proguard_map.txt'
+    keepFile = 'keep.txt'
+    jarFiles = ['lib1.jar', 'lib2.jar']
+    otherArgs = '' // any other command line options to `redex`
+    // see `redex --help` for details on these arguments
+}
+
 ```
-If you do not set the passes, all passes will be run by default. Sometimes you may not to run all optimisation passes, for example some appear to break when optimising kotlin code.
+If you do not set the passes or the config file, the default set of passes will
+be run. Sometimes you may not to run all optimisation passes, for example some
```

Comment:
May not want

Reply:
How would you prefer it to be written? I changed this because the old version isn't quite true anymore.

Reply:
Sorry, meant to say that "Sometimes you may not to run all" should be changed to "Sometimes you may not want to run all"

Reply:
Ah. Yeah. Of course. Good catch